prem_utils/models.json

```diff
@@ -1339,9 +1339,9 @@
                     ]
                 },
                 {
-                    "slug": "prem-1b-chat",
+                    "slug": "prem_1b_chat",
                     "model_type": "text2text",
-                    "coming_soon": true,
```

Comment:
why have we removed the coming_soon?

Reply:
Once released the coming_soon param becomes irrelevant. Followed the pattern of the other model definitions to just delete it and not make it False.

Do we wish to make this as `coming_soon: False`? That kinda seems redundant right?

Reply:
@vittoriop17 @giowe do we need the coming_soon?

Reply:
TLDR; I think it is correct to remove it if we wanna serve the model

@nsosio I think the flag is currently used to filter out these models and not show them in the platform. In one of the previous releases it was used to disabile the usage of the specified model (that was shown in the platform). But for clearness, I would wait @giowe's last word